### PR TITLE
Add guide: Monitor any process in real-time with Netdata

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

A new guide for monitoring processes by editing `apps_groups.conf` and finding the right charts in a dashboard.

I also made a few tweaks/improvements to the apps.plugin and ebpf.plugin docs along the way.

##### Component Name

area/docs
area/collectors

### CODE REVIEW

**Comment by joelhans** [comment]
[How to monitor any process with Netdata](https://trello.com/c/CLhK4It7/85-how-to-monitor-any-process-with-netdata)

**Comment by joelhans** [comment]
@ilyam8 || @thiagoftsm: Perhaps one of you two could give this a cursory review? It's touches on apps.plugin/ebpf.plugin capability, but mostly on a high level.

File: collectors/ebpf.plugin/README.md [hunk 1]
```diff
@@ -156,24 +156,27 @@ The eBPF collector also creates charts for each running application through an i
...
-    -   Number of process created trough `do_fork`
-    -   Number of threads created trough `do_fork` or `__x86_64_sys_clone`, depending on your system's kernel version.
-    -   Number of times that a process called `do_exit`. 
+
+-   eBPF file
+    -   Number of calls to open files. (`apps.file_open`)
+    -   Number of files closed. (`apps.file_closed`)
     -   Number of calls to open files that returned errors.
     -   Number of calls to close files that returned errors.
+-   eBPF syscall
```
Comment (ilyam8) [inline collectors/ebpf.plugin/README.md:165]: was changed in https://github.com/netdata/netdata/pull/10299

Comment (ilyam8) [reply collectors/ebpf.plugin/README.md:165]: oh, nvm, everything is ok 👍

File: docs/guides/monitor/process.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
+<!--
+title: Monitor any process in real-time with Netdata
+description: "Tap into Netdata's powerful collectors, with per-second utilization metrics for every process, to troubleshoot faster and make data-informed decisions."
+image: /img/seo/guides/monitor/process.png
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/guides/monitor/process.md
+-->
+
+# Monitor any process in real-time with Netdata
+
+Netdata is more than a heaping of generic system-level metrics and visualizations. Instead of providing only a bird's
```
Comment (aabatangle) [inline docs/guides/monitor/process.md:10]: I'm not sure 'heaping' can be used as a noun? It was a bit confusing/awkward for me.

File: docs/guides/monitor/process.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+# NETDATA processes accounting
+...
+```
+
+Restart Netdata with `service netdata restart`, or the appropriate method for your system, to start collecting
+utilization metrics from your application.
+
+## Visualize process metrics
+
+Now that you're colleting metrics for your process, you'll want to visualize them using Netdata's real-time, interactive
```
Comment (aabatangle) [inline docs/guides/monitor/process.md:202]: colleting --> collecting

File: docs/guides/monitor/process.md [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+
+### Using Netdata's application collector (`apps.plugin`)
+
+`apps.plugin` puts all of its charts under the **Applications** section of any Netdata dashboard.
+
+![Screenshot of the Applications section on a Netdata
+dashboard](https://user-images.githubusercontent.com/1153921/101401172-2ceadb80-388f-11eb-9e9a-88443894c272.png)
+
+Let's continue with the MySQL example. We can create a [test
+database](https://www.digitalocean.com/community/tutorials/how-to-measure-mysql-query-performance-with-mysqlslap) into
```
Comment (aabatangle) [inline docs/guides/monitor/process.md:218]: into --> in?

File: docs/guides/monitor/process.md [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+`apps.plugin` immediately collects and visualizes this activity `apps.cpu` chart, which shows an increase in CPU
+utilization from the `sql` group. There is a parallel increase in `apps.pwrites`, which visualizes writes to disk.
+
+![Per-application CPU utilization
+metrics](https://user-images.githubusercontent.com/1153921/101409725-8527da80-389b-11eb-96e9-9f401535aafc.png)
+
+![Per-application disk writing
+metrics](https://user-images.githubusercontent.com/1153921/101409728-85c07100-389b-11eb-83fd-d79dd1545b5a.png)
+
+Next, the `mysqlslap` utility queries the database to provide some real world-esque MySQL load.
```
Comment (aabatangle) [inline docs/guides/monitor/process.md:230]: This is a tiny detail, but I worry that users translating the docs with their browser might have trouble with language like 'real world-esque' or anything idiomatic.

File: docs/guides/monitor/process.md [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+
+> 💡 Click on any dimension below a chart in Netdata Cloud (or to the right of a chart on a local Agent dashboard), to
+> visualize only that dimension. This can be particularly useful in process monitoring to separate one process'
+> utilization from the rest of the system.
+
+### Using Netdata's eBPF collector (`ebpf.plugin`)
+
+Netdata's eBPF collector puts its charts in two places. Of most imporance to process monitoring are the **ebpf file**,
+**ebpf syscall**, **ebpf process**, and **ebpf net** sub-sections under **Applications**, shown in the above screenshot.
+However, you can also find
```
Comment (aabatangle) [inline docs/guides/monitor/process.md:249]: Did something get cut off here?

File: docs/guides/monitor/process.md [hunk 7]
```diff
@@ -0,0 +1,297 @@
...
+
+For example, running the above workload shows the entire "story" how MySQL interacts with the Linux kernel to open
+processes/threads to handle a large nubmer of SQL queries, then subsequently close the tasks as each query returns the
```
Comment (aabatangle) [inline docs/guides/monitor/process.md:252]: nubmer --> number

File: docs/guides/monitor/process.md [hunk 8]
```diff
@@ -0,0 +1,297 @@
...
+`ebpf.plugin`, and application-specific collectors to build targeted dashboards for monitoring key procesess across your
+infrastructure.
+
+Try running [Metric Correlations](https://learn.netdata.cloud/docs/cloud/insights/metric-correlations) on a node that's
+running the process(es) you're monitoring. Even if nothing is going wrong at the moment, Netdata Cloud's embedded
+intelligence helps you better understand how a MySQL database, for example, might influence a system's volume of memory
+page faults. And when an incident is afoot, use Metric Correlations to reduce mean time to resolution (MTTR) and
+cognitive load.
+
+If you want more specific metrics from your custom application, check out Netdata's [statd
```
Comment (aabatangle) [inline docs/guides/monitor/process.md:284]: statd --> statsd

**Review by aabatangle — APPROVED** [review]

File: docs/guides/monitor/process.md (hunk 6 above)
Comment (joelhans) [reply docs/guides/monitor/process.md:249]: Forgot to lop off that clause when I moved the sentence to the end of the section!

**Review by ilyam8 — APPROVED** [review]

**Review by aabatangle — APPROVED** [review]

**Pull request merged**